packages/e2e/example/test_driver/example_e2e_test.dart

```diff
@@ -2,4 +2,4 @@ import 'dart:async';
 
 import 'package:e2e/e2e_driver.dart' as e2e;
 
-Future<void> main() async => e2e.main();
+Future<void> main() async => e2e.e2eDriver();
```

Comment:
Why this change?

Reply:
It's in the example. I'm thinking with the new function `e2e.e2eDriver` is the recommended solution, but `main` is preserved to avoid breaking change.

Reply:
But why even change the method here? Why not just add a new test that covers the case you're interested in?

Reply:
I thought these examples are not test cases but demos for users about how to use e2e.

Reply:
And another thing is, we currently don't have a `test` directory under e2e, I'm not sure how it's unit tested, and this part have to connect to a driver and I don't even know if we are able to run this repo on devicelab.

Reply:
We could certainlyl add tests to e2e :)

It's just not clear to me why we're adding this redirecting method and calling it directly instead of just calling main.

Reply:
So `main` is limited for the format of arguments, due to convention and due to if we want `e2e_driver.dart` to be used directly as the driver script. `main` is using the default argument version of `e2eDeiver` but I'm not expecting future users to use `main` anymore.

Reply:
@liyuqian No it shouldn't. On the one hand, It's just an async main, there's nothing it need to await for, on the other hand, return a Future without awaiting is very common in Framework `Future<void> Foo() {return Future<void>.delayed(...)`, where whoever `await` this `Foo` would actually `await` this `Future.delayed`.

Reply:
Oh right, it's ` => e2e.main()` instead of `{ e2e.main(); }`. I think only the latter needs `{ await e2e.main(); }`. Anyway, if we don't `await`, can we remove the `async` and just use `Future<void> main() => e2e.main();`? That way, it's very clear that no `await` is expected there.

Reply:
> Oh right, it's ` => e2e.main()` instead of `{ e2e.main(); }`. I think only the latter needs `{ await e2e.main(); }`. Anyway, if we don't `await`, can we remove the `async` and just use `Future<void> main() => e2e.main();`? That way, it's very clear that no `await` is expected there.

Good point! I deleted the `async`.